frontend/wallet/src/render/components/Footer.re

```diff
@@ -82,15 +98,7 @@ module RightButtons = {
 
 [@react.component]
 let make = (~stakingKey, ~settings) => {
-  <div
-    className=Css.(
-      style([
-        display(`flex),
-        justifyContent(`spaceBetween),
-        alignItems(`center),
-        padding(`rem(2.0)),
-      ])
-    )>
+  <div className=Styles.footer>
```

Comment:
@Schmavery and I ended up preferring inlining of styles as we were working on the codaprotocol.com redesign. And only using variables when styles were taken from the styleguide or reused more than once. See https://github.com/CodaProtocol/coda/blob/master/frontend/website/src/HeroSection.re#L208 for an example. This approach made it easier for us to quickly tweak styles as we didn't have to jump up and down files since style and layout were right next to each other.

Do you have a different opinion on this?

Reply:
I think it's probably better to use a Styles module so it's easy to see all the styles at the top of the file and allows us to only use one syntax for writing styles. Also gives us extensibility if we ever want to (i.e. `merge([Header.Styles.header, customStyles])`

Reply:
I'd argue it's a lot harder to see what the styles are when they're not in the context of the elements onto which they're applying. What do you mean by one syntax for writing styles?

Reply:
okay let's try it